### ISSUE TITLE
Filetypes-handled option doesn't prevent loading on Rmd files

### ISSUE THREAD

**Issue by mmclaren42** [vim-pandoc/vim-pandoc#277]
I would like to use vim-pandoc (and vim-pandoc-syntax) on markdown files but not R-markdown files. I have tried the below options based on the help page "vim-pandoc-filetypes" and in all cases vim-pandoc still loads on R-markdown files (`ft=rmd`).
```
let g:pandoc#filetypes#handled = ["pandoc", "markdown"]
let g:pandoc#filetypes#handled = ["markdown"]
let g:pandoc#filetypes#handled = []
```
Sorry if I'm missing something simple, and thanks for the plugin!

**Comment by fmoralesc** [comment]
Do you have the `vim-rmarkdown` plugin installed? That would load vim-pandoc too.

**Comment by mmclaren42** [comment]
No, I do not use vim-rmarkdown, but I do use Nvim-R. However, even with Nvim-R disabled (placed into the opt/ folder of native plugin management), vim-pandoc still loads on Rmd documents.
